src/Components/Milestones/Assessment.js

```diff
@@ -1,106 +1,88 @@
+  });
+
+  return keyValueMap;
+}
 
-function Assessment() {
+function Assessment({ jobseeker }) {
   const [experiences, setExperience] = useState([]);
   const [industries, setIndustries] = useState([]);
   const [skills, setSkills] = useState([]);
-  const store = useSelector((state) => state.auth.value);
 
   useEffect(() => {
     const start = async () => {
-      const jobseekerData = await fetchJobseekerData(store.email);
-      console.log(jobseekerData.data());
-      console.log(objToArray(jobseekerData.data().industryInterest));
-
       const updatedExperience = [];
-      industryInterestOptions.forEach((experience) => {
-        updatedExperience.push({ label: experience, value: false, bullets: [] });
-      });
-
-      updatedExperience.forEach((experience, i) => {
-        if (objToArray(jobseekerData.data().industryInterest).includes(experience.label)) {
-          updatedExperience[i].value = true;
-        }
+      jobseeker.skillsChecklist.forEach((experience) => {
+        updatedExperience.push({ label: experience, value: true, bullets: [] });
       });
 
       const updatedIndustryInterests = [];
-      industryInterestOptions.forEach((industry) => {
-        updatedIndustryInterests.push({ label: industry, value: false, bullets: [] });
-      });
-
-      updatedIndustryInterests.forEach((industry, i) => {
-        if (objToArray(jobseekerData.data().industryInterest).includes(industry.label)) {
-          updatedIndustryInterests[i].value = true;
-        }
+      jobseeker.industryInterest.forEach((industry) => {
+        updatedIndustryInterests.push({ label: industry, value: true, bullets: [] });
       });
 
       const updatedSkills = [];
-      skillsChecklistOptions.forEach((skill) => {
-        updatedSkills.push({ label: skill, value: false, bullets: [] });
+      jobseeker.generalSkills.forEach((skill) => {
+        updatedSkills.push({ label: skill, value: true, bullets: [] });
       });
 
-      updatedSkills.forEach((skill, i) => {
-        if (objToArray(jobseekerData.data().skillsChecklist).includes(skill.label)) {
-          // update.bullets
-          updatedSkills[i].value = true;
-        }
-      });
       setExperience(updatedExperience);
       setIndustries(updatedIndustryInterests);
       setSkills(updatedSkills);
-      console.log(updatedIndustryInterests);
     };
     start();
   }, []);
 
+  const dataSets = [
+    { data: jobseeker.degrees, label: 'Degree' },
+    { data: jobseeker.certificates, label: 'Certificate' },
+  ];
+
   return (
     <div className="assessment">
       <h6 className="contentTitle">Client Info</h6>
       <hr className="shortLine" />
-      <MilestoneClientInfo data={mockData} />
+      <MilestoneClientInfo data={jobseeker.clientInfo} />
       <h6 className="contentTitle">Education Info</h6>
       <hr className="shortLine" />
+      { dataSets.map((dataSet) => (
+        <MilestoneClientInfo data={mapArrays(dataSet.data, dataSet.label)} />
```

Comment:
whenever you map through elements like this, it's good to give each element a key! this is so the DOM can distinguish between all the nearly identical components. if there's any part of the dataSet that's unique you can use that as the key, or you can import the uuidv4 library!
eg add this library
import { v4 as uuidv4 } from 'uuid';

and then
```suggestion
        <MilestoneClientInfo key={uuidv4()} data={mapArrays(dataSet.data, dataSet.label)} />
```